meetings/2022-03-08.md

```diff
@@ -0,0 +1,114 @@
+
+* Justin: Last we decided on renaming Access Consent and Data Consent. I should get it done by next session. I'm also updating sai-java code as I do it.
+
+#### [dataOwner in scope:All]( https://github.com/solid/data-interoperability-panel/issues/248)
+
+* Justin: We have incorrect snippet with dataOwner on All scope. It is super minor I can get it done unless someone else will get to it first.
+
+#### [Finalize design for trusted consents and trusted grants](https://github.com/solid/data-interoperability-panel/issues/187)
+
+* Justin: We need to make PR to the spec. It should represent consent given to trusted agents and applications. This would apply to Authorization Agent or Social Agent you delegated administrative access to. For example admin of an organization.
+* Wouter: What exactly do we understand under trusted agents in that sense?
+* Justin: Trusted agent could be a Social Agent or an Application. There have been given essentially administrative power and capability over the data owner's registry. Could apply to Access Registry, the Agent Registry or Data Registry.
+* Wouter: How does that align with earlier advise not to have more than one Authorization Agent?
+* Justin: Authorization Agent is a trusted agent but not all trusted agents are AAs.
+* Wouter: Okay, so there will be different tiers of permissions within trusted grants?
+* Pavlik: at the moment we've just discussed full admin tier, but we may want to be more granular. should work over use cases. Also did we capture more generic data access (e.g. data browser)
+* Justin: That discussion I put it under next header, proposal for publicly accessible data.
+
+#### Generic data grants
+
+* Pavlik: mostly applies to data registries, e.g. in pod browser case where something should have access to more than a specific shape tree
+* Justin: Few use case would be a good place to start. 
+* Pavlik: In the spirit of discussion with Jeff last week, look at how solidOS and other apps can more readily adopt interop patterns
+ 
+#### Proposal for publicly accessible data
+
+[Issue#243](https://github.com/solid/data-interoperability-panel/issues/243)
+
+* Justin: We need to build on what Laurens put in that issue.
+* Laurens: I can propose something there based on what's in the spec
+
+#### [Incorporate profile requirements from solid-oidc](https://github.com/solid/data-interoperability-panel/issues/210)
+
+* Justin: I actually think we have any blockers left here. In sai-java i have it implemented. This is a matter of reflecting it in the spec. We have now JSON-LD context for the spec itself and patter for how those contexts can be combined.
+* Pavlik: I think we should consider again whether we want to mandate that context because we don't guarantee on solid that the compacted version is preserved and it becomes more complicated to store it. I think it's problematic that it couldn't be reliably stored in solid storage.
```

Comment:
```suggestion
* Pavlik: I think we should consider again whether we want to mandate that context, because we don't guarantee on solid that the compacted version is preserved, and it becomes more complicated to store it. I think it's problematic that it couldn't be reliably stored in solid storage.
```